book/module1/2 video comp thinking.md

```diff
@@ -0,0 +1,27 @@
+# Video: Computational thinking
```

Comment:
`git pull origin main` and move the content of this file to 2-videos-...

Reply:
move to 2-videos-variables.md?